style.css

```diff
@@ -136,3 +136,105 @@ body {
   align-items: center;
   justify-content: space-around;
 }
+
+/* Work section */
+#sct2 { margin-top: 83px; }
+
+.work-list {
+  width: 100%;
+  height: auto;
+  display: grid;
+  grid-template-columns: auto;
+  padding: 0 24px;
+}
+
+.work-list-item {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 40px;
+}
+
```

Comment:
- [ ] Please fix the alignment your mobile version is not aligned properly. 👉 [your project](https://user-images.githubusercontent.com/61562626/129043476-b0c6a4a2-d375-465d-b8b0-4cafec2e9d98.png), and original [see the link here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)